templates/data/generic_external_module_data.yaml

```
@@ -46,6 +46,26 @@ templates:
                   global:
                     - jobConfig_default
                     - jobConfig_presubmit
+              - jobConfig:
+                  labels:
+                    preset-sa-kyma-push-images: "true"
+                  env:
+                    KUSTOMIZE_VERSION: "v4.5.6"
+                    MODULE_REGISTRY: "europe-west4-docker.pkg.dev/sap-se-cx-kyma-wookiee/dev-registry/unsigned" 
+                    MODULE_VERSION: "0.0.${PULL_NUMBER}"
+                    IMG: "eu.gcr.io/kyma-project/keda-manager:PR-${PULL_NUMBER}"
+                  run_if_changed: "^operator/(go.mod|go.sum)$|^operator/*/(.*.go|Makefile|.*.sh)"
+                  name: pull-make-module-build
```

```suggestion
                  name: pull-make-module-build
                  privileged: false
```
disable privileged mode on job-level.